LCLSGeneral/LCLSGeneral/POUs/Data/FB_EpicsCentroidMonitor.TcPOU

```diff
@@ -0,0 +1,223 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_EpicsCentroidMonitor" Id="{b49e72a7-c7c2-451d-96d1-aade68e9ce98}" SpecialFunc="None">
+    <Declaration><![CDATA[(*
+    EPICS AreaDetector Stats Plugin Centroid Monitor
```

Comment:
Just curious: what's the application for this FB? Seems neat but my imagination can't quite grasp what it's supposed to be used for. All the logic seems correct but without the context I'm not sure what else to look closely at.